util/expand-liquid.js

```diff
@@ -59,7 +58,7 @@ async function directoryNames(parentPath) {
     .map(dirent => dirent.name);
 }
 
-async function expandExtensionLiquidTemplates(domainName) {
+async function expandExtensionLiquidTemplates(domainName, jsFlavor) {
```

Comment:
Should we support `flavor` more generally instead of `jsFlavor`? We could keep a mapping of `flavor` to file extensions.

Reply:
Renamed to `flavor` to be consistent with what the CLI does. But for know the file extensions only matter for `javascript` as language because it is the only language with liquid templates that don't have file extensions